frontend/src/components/AdminControls/Users.js

```diff
@@ -18,6 +21,26 @@ const Users = ({ users, refreshUsers }) => {
     setSuspendModal(!suspendModal);
   };
 
+  const togglePromoteModal = () => {
+    setPromoteModal(!promoteModal);
+  };
+
+  const promoteButtonFunction = (userId, isAdmin) => {
```

Comment:
NitPick: Since this function is more-or-less an event handler, I renaming it to something like `handlePromote` would make sense.